pwa/src/templates/objectTemplate/ObjectTemplate.tsx

```diff
@@ -6,10 +6,18 @@ import { Container } from "@conduction/components";
 import { faPlus } from "@fortawesome/free-solid-svg-icons";
 import { Button } from "../../components/button/Button";
 import { OverviewPageHeaderTemplate } from "../templateParts/overviewPageHeader/OverviewPageHeaderTemplate";
-import { ObjectsTableTemplate } from "../templateParts/objectsTable/ObjectsTable";
+import { ObjectsTable } from "../templateParts/objectsTable/ObjectsTable";
+import { useObject } from "../../hooks/object";
+import { useObjectsStateContext } from "../../context/objects";
+import Skeleton from "react-loading-skeleton";
 
 export const ObjectTemplate: React.FC = () => {
   const { t } = useTranslation();
+  const { objectsState } = useObjectsStateContext();
+  const [searchQuery, setSearchQuery] = React.useState<string>("");
```

Comment:
Feedback goes for other implementations as well.

We're never actually setting the `searchQuery` resulting in the search not working anymore. We're keeping track of `searchQuery` (and setting it) in the table component as well, but the updated value is never updated within the react query key and therefore not being filtered.

Possible solution:

1. Passing `searchQuery` and `setSearchQuery` to the object table and no longer initiate it within the object table (so effectively, when we set the search query in the object table, we're actually updating the one in the implementation layer).